docs/reference/builder.md

```diff
@@ -1561,6 +1562,11 @@ is considered to have failed.
 It takes **retries** consecutive failures of the health check for the container
 to be considered `unhealthy`.
 
+Given a **start period** the tries performed during that period will not be
```

Comment:
**start-period** provides initialization time for containers that need time to bootstrap. Probe failure during ...